ephios/core/templates/core/shift_copy.html

```diff
@@ -0,0 +1,24 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}
+{% load static %}
+{% load i18n %}
+
+{% block title %}
+    {% translate "Copy shift" %}
+{% endblock %}
+
+{% block css %}
+{% endblock %}
+
+{% block content %}
+    <div class="page-header">
+        <h1>
+            {% translate "Copy shift" %} "{{ shift.event.title }}" <small>({{ shift.get_datetime_display }})</small>
+        </h1>
+    </div>
+    {{ form.errors }}
```

Comment:
in both event_copy and shift_copy, these should be in an alert box like we do for other forms, right?